business/definitionService.js

```diff
@@ -88,6 +88,35 @@ class DefinitionService {
     return sortedUniq([...harvest, ...curated])
   }
 
+  /**
+   * Get a list of all the definitions that exist in the store matching the given coordinates
+   * @param {Object[]} coordinatesList
```

Comment:
Can this be an array of EntityCoordinates? seems like a better API and the code in here does not have to do the  `fromObject` calls

Reply:
This method is the same used in #288 , we will update it from there

Reply:
Right. that one looks more up to date but still deals in object. I'm guessing that this branch is based on that branch or something?

Reply:
Not completely, we used only this method, and actually in a previous implementation.
So we're gonna updated it directly in the code.